code.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "repodive-tools",
+  "description": {
+    "en": {
+      "shortDescription": "Scripts and tools to gather information about git repositories.",
+      "longDescription": "This repository is a collection of scripts and tools for a given repodiving effort. Repodiving in this context means going through a git repository and gathering relevant information for a specific purpose."
+    }
+  },
+  "status": "archival",
+  "permissions": {
+    "licenses": [
+      {
+        "URL": "LICENSE",
+        "name": "CC0-1.0"
+      }
+    ]
+  },
+  "organization": "Centers for Medicare & Medicaid Services",
+  "repositoryURL": "https://github.com/DSACMS/repodive-tools",
+  "vcs": "git",
+  "laborHours": null,
+  "platforms": [
+    "mac"
+  ],
+  "categories": [
+    "data-collection, data-analytics"
```

Comment:
Oh nooooo this is formatted incorrectly, I think this is a code.json bug. You can fix manually for now but we should def file a ticket to fix this on repo-scaffolder